hack/tools/go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/openshift/cluster-monitoring-operator/hack
 go 1.14
 
 require (
-	github.com/brancz/gojsontoyaml v0.0.0-20191212081931-bf2969bbd742
+	github.com/brancz/gojsontoyaml v0.0.0-20200602132005-3697ded27e8c
 	github.com/campoy/embedmd v1.0.0
 	github.com/google/go-jsonnet v0.16.1-0.20200703153429-aaf50f5b655f
 	github.com/jsonnet-bundler/jsonnet-bundler v0.4.0
-	github.com/prometheus/prometheus v1.8.2-0.20200329163539-7646cbca3282
+	github.com/prometheus/prometheus v1.8.2-0.20201015110737-0a7fdd3b7696
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
@simonpasquier the gopkg.in/yaml.v2 v2.1.1 dependency of gojsontoyaml was broken by the prometheus upgrade for some reasons.
Upgrading gojsontoyaml and its transitive dependency solve the issue.

Reply:
Ok I found the issue, it's because the YAML formatting has changed between v2.2.1 and v2.3.0. And it has been reverted back in v2.4.0 (see https://github.com/go-yaml/yaml/commit/7649d4548cb53a614db133b2a8ac1f31859dda8c) because of complaints from the kubernetes community. In general, I'd say we don't need to worry about it and just use whatever version of `gopkg.in/yaml.v2` is required by our deps. But A PR to https://github.com/brancz/gojsontoyaml to bump to v2.4.0 is probably the good approach.

Reply:
OK, thank you Simon for the insight! :100:. I'll prepare a PR against gojsontoyaml to bump to v2.4.0